### DISCUSSION TITLE
Can you set index of carousel or just read it?

### DISCUSSION THREAD

**Discussion by austinmrobinson** [dscyrescotti/ShuffleIt#71] [Q&A] [answered]
I am trying to let users skip to a certain item within the carousel by choosing it from a list. However, it doesn't look like I can _set_ the index of the carousel, but only read it. Is that true?

I have used the `carouselPublisher.send()` method, but it does not let me jump to a specific item

**Comment by dscyrescotti** [comment, marked as answer]
@AustinMRobinson Yes, you're right. Sadly, you cannot set a specific index to jump to its respective item at the moment. However, you can jump to the left or right from the current item using `carouselTrigger(_:)` modifier. I also note your requirement and will make it include in future. Anyway, thank you for letting me know. :)
